spec.html

```diff
@@ -30112,7 +30134,7 @@ <h1>String.prototype.matchAll ( _regexp_ )</h1>
           1. Let _O_ be ? RequireObjectCoercible(*this* value).
           1. If _regexp_ is neither *undefined* nor *null*, then
             1. Let _isRegExp_ be ? IsRegExp(_regexp_).
-            1. If _isRegExp_ is *true*, then
+            1. If _isRegExp_ is true, then
```

Comment:
cc @bakkot for the linter

Reply:
Added a desiderata to [the list](https://github.com/tc39/ecmarkup/issues/173).

That one is a little tricky to lint for, because we say stuff like "the substeps are only applied if the predicate is true" or "if one or more of the following conditions is true:".